docs/docs/cmd/teams/meeting/meeting-transcript-list.md

```diff
@@ -0,0 +1,93 @@
+# teams meeting transcript list
+
+Lists all transcripts for a given meeting
+
+## Usage
+
+```sh
+m365 teams meeting transcript list [options]
+```
+
+## Options
+
+`-u, --userId [userId]`
+: The id of the user, omit to list meeting transcripts list for current signed in user. Use either `id`, `userName` or `email`, but not multiple.
+
+`-n, --userName [userName]`
+: The upn of the user, omit to list meeting transcripts list for current signed in user. Use either `id`, `userName` or `email`, but not multiple.
+
+`--email [email]`
+: The email of the user, omit to list meeting transcripts list for current signed in user. Use either `id`, `userName` or `email`, but not multiple.
+
+`-m, --meetingId <meetingId>`
+: The id of the meeting.
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Lists all transcripts made for the current signed in user and Microsoft Teams meeting with given id
+
+```sh
+m365 teams meeting transcript list --meetingId MSo1N2Y5ZGFjYy03MWJmLTQ3NDMtYjQxMy01M2EdFGkdRWHJlQ
+```
+
+Lists all transcripts for a meeting of a specific user
+
+```sh
+m365 teams meeting transcript list --userName garthf@contoso.com --meetingId MSo1N2Y5ZGFjYy03MWJmLTQ3NDMtYjQxMy01M2EdFGkdRWHJlQ
+```
+
+## Remarks
+
+!!! attention
+    This command is based on a Microsoft Graph API that is currently in preview and is subject to change once the API reached general availability.
+
```

Comment:
Added an extra remark that when you are using application permissions, you should create and extra access policy.